IgOmeProfiling_pipeline.py

```diff
@@ -18,6 +19,12 @@ def run_pipeline(fastq_path, barcode2samplename_path, samplename2biologicalcondi
                  allowed_gap_frequency, concurrent_cutoffs, meme_split_size, number_of_random_pssms,
                  rank_method, tfidf_method, tfidf_factor, shuffles,
                  run_summary_path, error_path, queue, verbose, argv):
+    
+    # check the validation of files barcode2samplename_path and samplename2biologicalcondition_path
+    check_files = is_validation_files(samples2bc_path=samplename2biologicalcondition_path,barcode2samples_path=barcode2samplename_path)
```

Comment:
`check_files` is not a  good name, the function returns if the files are valid and not if we should check the files.
Also, we should differentiate between the different errors and log them correctly.
We don't want the user to guess which file is invalid and why